app/cells/markdown_readonly/markdown_readonly.coffee

```diff
@@ -0,0 +1,7 @@
+#= require jquery
```

Comment:
The idea here was to not require the dependencies globally anymore but have every component require their own dependencies. If there are any dependencies not required by any component, they are not included. That way we could even include only selected components and their dependencies in a project. In order to do this in a way that we don't just forget about including the right dependencies and having it break if the component actually is included individually, we would need a feature test for each component, only including that one component. This is another thing I would rather do separately.

tl;dr: I'll remove it for now. ;)